doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -205,6 +208,28 @@ def plot_comparison(plot1, plot2, title1, title2):
 
 #####################################################################
 # Let us visualize one restored image, where the dark spots have been
-# inpainted.
+# inpainted. First, we find the contours of the dark spots (i.e., of the mask)
+# so we can draw them on top of the restored image;
+
+contours = measure.find_contours(mask_dilate)
+
+# Gather all (row, column) coordinates of the contours
+x = []
+y = []
+for contour in contours:
+    x.append(contour[:, 0])
+    y.append(contour[:, 1])
+# Flatten them
+x_flat = np.concatenate(x).ravel()
+y_flat = np.concatenate(y).ravel()
+# Create a sparse matrix corresponding to the mask of these contours
+data = np.ones(x_flat.shape, dtype='bool')
+mtx = sparse.coo_matrix((data, (x_flat, y_flat)), shape=mask_dilate.shape)
+# Convert it to array
+arr = mtx.toarray().astype('float')
+
+fig = px.imshow(image_seq_inpainted[50], color_continuous_scale='gray')
+fig.add_trace(go.Contour(z=arr, contours_coloring='lines'))
```

Comment:
```suggestion
fig.add_trace(go.Contour(z=arr, contours_coloring='lines', showscale=False))
fig.update_layout(height=500, width=500, coloraxis_showscale=False)
```
Clean up the plot

Reply:
Let's reflect on the `height=500, width=500` suggestion, found in the other Plotly figures... I actually find them really small. It strikes me even more with the contours of dark spots: Unless you zoom in, it's not even clear that they're contours!

It doesn't always make sense to constrain in height: I understand that the image is square, so maybe you want to preserve the aspect ratio, but since this height/wiidth include the legend, time slider, etc. I think we should either constrain only in width (at 600 or 700?), or drop this altogether. There are no such constraints [here](https://scikit-image.org/docs/dev/auto_examples/applications/plot_fluorescence_nuclear_envelope.html) and the rendering seems responsive enough... What do you think, @ana42742 and @decorouz?

Reply:
The thought behind setting the height and width arguments is to ensure consistency of figure dimension. Upon reviewing [this](https://scikit-image.org/docs/dev/auto_examples/applications/plot_fluorescence_nuclear_envelope.html), I will favor dropping both arguments. @ana42742 your thoughts?

Reply:
I am firmly in the camp that plotting APIs are confusing enough already so any parameter we don't have to explicitly specify reduces the complexity. If the default size works use that. I am not sure there is a inherent benefit in having all figures the same size when not directly next to each other.

Reply:
> If the default size works use that.

I played around a bit but my conclusion about the rendering with default sizes isn't firm... ("it depends?"). I still went ahead with d72ac60f307.